app/src/main/res/values/strings.xml

```diff
@@ -121,12 +121,12 @@
     <string name="intent_import_card_from_url_path_prefix" translatable="false">/Catima/share</string>
     <string name="intent_import_card_from_url_host_old" translatable="false">brarcher.github.io</string>
     <string name="intent_import_card_from_url_path_prefix_old" translatable="false">/loyalty-card-locker/share</string>
-    <string name="importSuccessful">Successfully imported loyalty card data</string>
-    <string name="exportSuccessful">Successfully exported loyalty card data</string>
+    <string name="importSuccessful">Card data imported</string>
+    <string name="exportSuccessful">Card data exported</string>
 
     <string name="enter_group_name">Enter group name</string>
     <string name="groups">Groups</string>
-    <string name="noGroups">You don\'t have any groups at the moment. Click the "+" (plus) button to get started.\n\nCards can be assigned to groups to make them easier to find.</string>
+    <string name="noGroups">Click the "+" (plus) button to add groups first.\n\nCards can be assigned in groups to make them easier to find.</string>
```

Comment:
I'm pretty sure "assigned in" is not correct grammar. Either "assigned to" or "put in" maybe.

Reply:
https://idioms.thefreedictionary.com/assigned+to Just technically correct isn't really good though.
Was thinking "can be grouped".

Reply:
"can be grouped" sounds good to me

Reply:
Should it be "entries", as tickets are also a thing now?

Reply:
Well they're not a thing yet, #69 hasn't been merged yet (other things have been taking priority). Not sure what would be best to do here. This is displayed in the groups screen so it would most likely be confusing to use "Entries" here.